src/satosa/micro_services/attribute_generation.py

```diff
@@ -0,0 +1,145 @@
+import re
+import pystache
+
+from .base import ResponseMicroService
+
+def _config(f, requester, provider):
```

Comment:
I was planning to do that in a later refactor - dunno if I can cherry-pick that function